docs(repo): update some conformance docs (#29073)
nrwl/nx: docs/shared/features/powerpack/conformance.md

}
```

> ## Enforce Rules with the `nx conformance` and `nx conformance:check` commands

Reviewer: ```suggestion
## Enforce Rules with the `nx conformance` and `nx conformance:check` Commands
```

> The `@nx/powerpack-conformance` plugin enables the `nx conformance` and `nx conformance:check` commands which checks all the configured rules. The difference is that `nx conformance` will invoke any fix generators on a configured rule automatically, whereas `nx conformance:check` will only check the current workspace state and provide violations if applicable.

Reviewer: ```suggestion
The `@nx/powerpack-conformance` plugin enables the `nx conformance` and `nx conformance:check` commands which check all the configured rules. The difference is that `nx conformance` will invoke any fix generators on a configured rule automatically, whereas `nx conformance:check` will only check the current workspace state and show any violations.
```

> Therefore, `nx conformance` is intended to be run locally while working on a feature branch, be added to the beginning of your CI process so that the conformance rules are enforced for every PR.

Reviewer: ```suggestion
Therefore, `nx conformance` is intended to be run locally while working on a feature branch. `nx conformance:check` should be added to the beginning of your CI process so that the conformance rules are enforced for every PR.
```
